.github/workflows/validate-pr.yml

```diff
@@ -35,27 +35,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
-      - name: Cache Gatsby cache folder
+      - name: Cache Gatsby build
         uses: actions/cache@v2
         with:
-          path: .cache
-          key: ${{ runner.os }}-cache-gatsby-${{ github.sha }}
+          path: |
+            public
+            .cache
+          key: ${{ runner.os }}-gatsby-build-${{ github.run_id }}
           restore-keys: |
-            ${{ runner.os }}-cache-gatsby-
-
-      - name: Gatsby public folder
-        uses: actions/cache@v2
-        with:
-          path: public
-          key: ${{ runner.os }}-cache-public-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-cache-public-
+            ${{ runner.os }}-gatsby-build-
 
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
       - name: Gatsby build
-        run: yarn build:production
+        run: yarn build:production --log-pages
+        env:
+          GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES: true
```

Comment:
were you able to get any measurement on what the difference in build changes was 😄 ? Just out of curiosity

Reply:
Not quite unfortunately 😢 . It deleted the cache on the last push. Figured we'd try it out on `develop` and see how it goes.